### PR TITLE
Fat 5517 create producer crud

### PR DESCRIPTION
**Pull request by Activict**

### CODE REVIEW

File: StoreApp/Controllers/ProducersController.cs [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+                return RedirectToAction("Index");
+            }
+
+            producerService.Delete(producer.Id);
+
+            TempData["Message"] = "Producer deleted success!";
+
+            return RedirectToAction("Index");
+        }
+
```
Comment (AleksandroBoko) [inline StoreApp/Controllers/ProducersController.cs]: Unnecessary space

File: StoreApp.BLL/Services/ProducerService.cs [hunk 2]
```diff
@@ -16,13 +16,15 @@ public class ProducerService : IProducerService
 
         public ProducerService(IUnitOfWork uof)
         {
-            config = new MapperConfiguration(cfg => cfg.CreateMap<Producer, ProducerDTO>()).CreateMapper();
+            config = new MapperConfiguration(cfg => { cfg.CreateMap<Producer, ProducerDTO>();
```
Comment (AleksandroBoko) [inline StoreApp.BLL/Services/ProducerService.cs]: Btw if we keep this code here we should format it

File: StoreApp.BLL/Services/ProducerService.cs [hunk 3]
```diff
@@ -16,13 +16,15 @@ public class ProducerService : IProducerService
...
+                                                      cfg.CreateMap<ProducerDTO, Producer>(); }).CreateMapper();
             DataBase = uof;
         }
 
         public ProducerService()
         {
-            config = new MapperConfiguration(cfg => cfg.CreateMap<Producer, ProducerDTO>()).CreateMapper();
+            config = new MapperConfiguration(cfg => { cfg.CreateMap<Producer, ProducerDTO>();
```
Comment (AleksandroBoko) [inline StoreApp.BLL/Services/ProducerService.cs]: The same

File: StoreApp/Controllers/ProducersController.cs [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+        public ProducersController()
+        {
+            producerService = new ProducerService();
+
+            config = new MapperConfiguration(cfg =>
+            {
+                cfg.CreateMap<ProducerDTO, ProducerViewModel>();
+                cfg.CreateMap<ProducerViewModel, ProducerDTO>();
+            }).CreateMapper();
```
Comment (AleksandroBoko) [inline StoreApp/Controllers/ProducersController.cs:23]: We can use new mapper

File: StoreApp.BLL/Services/ProducerService.cs (hunk 2 above)
Comment (Activict) [reply StoreApp.BLL/Services/ProducerService.cs]: fixed

File: StoreApp.BLL/Services/ProducerService.cs (hunk 3 above)
Comment (Activict) [reply StoreApp.BLL/Services/ProducerService.cs]: fixed

File: StoreApp/Controllers/ProducersController.cs (hunk 4 above)
Comment (Activict) [reply StoreApp/Controllers/ProducersController.cs:23]: after discussion, left as is

File: StoreApp/Controllers/ProducersController.cs (hunk 1 above)
Comment (Activict) [reply StoreApp/Controllers/ProducersController.cs]: fixed

**Pull request merged**